build.gradle

```diff
@@ -17,7 +17,7 @@ buildscript {
 
 allprojects {
     group = "me.liuwj.ktorm"
-    version = "2.6"
+    version = "2.7-SNAPSHOT"
 }
```

Comment:
Please also modify the `build.gradle` file, add your GitHub ID to the developer info: 

```kotlin
developers {
    developer {
        id = "vincentlauvlwj"
        name = "vince"
        email = "me@liuwj.me"
    }
    developer {
        id = "waluo"
        name = "waluo"
        email = "1b79349b@gmail.com"
    }
}
```